app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt

```diff
@@ -223,6 +223,8 @@ class TypingGameActivity : GameActivity() {
         findViewById<LinearLayout>(R.id.displayGuess).removeAllViews()
         findViewById<EditText>(R.id.yourGuessET).text.clear()
         findViewById<EditText>(R.id.yourGuessET).isEnabled = true
+        val score = gameManager.getCorrectSongs().size
+        findViewById<TextView>(R.id.playerScore).text = "@string/your_score_dot_dot $score"
```

Comment:
```suggestion
        findViewById<TextView>(R.id.playerScore).text = "@string/your_score_dot_dot $score"
```
Great changes 😃 
Small comment:
- Can you change the string name according to the convention here [https://github.com/MaximeZmt/SDP_2022-Vibester/wiki] (We try to keep things ordered for new string like that 😉) It would be typingGame_yourScore (and can also change its place in the string.xml at the top of the file in the section if it exists or create TypingGame)